---
layout: page
title: About
permalink: /about/
---

### We can do better...
> "...it would appear to be a pointless and doubtful exercise to examine and
> disentangle the apparently random appearing bacteria in normal feces and the
> intestinal tract, a situation that seems controlled by a thousand
> coincidences... Yet I have nevertheless devoted myself now for a year virtually
> exclusively to this special study, it was with the conviction that the
> accurate knowledge of these conditions is essential, for the understanding of
> not only the physiology of digestion..., but also the pathology and therapy of
> microbial intestinal diseases."
>~~Theodore Escherich, 1885

Nearly 130 years ago, the pioneer of the modern movement to understand the
microbiome operated in ignorance of evolutionary theory, genomics, ecology,
statistics, or the knowledge that most bacteria cannot be cultured in the
laboratory. Today, even with amazing advances in all of these fields
microbiologists are still grappling with the questions that motivated Escherich
to understand why some of his pediatric patients developed diarrhea and others
did not. In many ways it is amazing how far the field has developed since
Escherich isolated *Communialis coli*, yet it is also distressing that in spite of
these advances we are still struggling to address fundamental questions about
the interplay between humans and the microbes that reside on and within us - the
human microbiome. The Schloss lab is motivated by the opportunities presented by
combining the tools of microbiology, ecology, and bioinformatics to address many
of these fundamental and applied questions.


### The human microbiome
The human microbiome is more than ten-time larger than the number of human cells
and represents a collective genome that is at least 100-times more complex than
our genome. Although many people are interested in pathogens, until recently,
there has been little emphasis placed on "the good guys" or determining how our
microbiome keeps us healthy. Numerous lines of evidence indicate that our
genetics, diet, behavior, and life experiences affect the structure of our
microbiome and make us each unique. Yet, because of this daunting intra- and
inter-personal complexity, it is difficult to make statements about the precise
interactions between specific bacterial groups within our microbiome and their
interaction with overall human health.


### Dynamics of the human microbiome in health and disease
My lab is specifically interested in understanding the temporal stability of the
microbiome and how perturbations in the microbiome (i.e. dysbiosis) relate to
changes in health and disease. The overall hypothesis of the lab is that each
individual is their own indication of "normal" and deviations from this state
will tell us a lot about our state of health. To this end we are pursuing two
projects, which simulate a personal medicine strategy using the murine
microbiome. First, we are taking a classical ecology approach to study the
effect of various perturbations on community stability. Through this approach we
can determine the relationship between diversity and stability and the
resistance and resilience of the microbiome to antibiotic treatments. Second, we
are using mouse models of tumor formation in the colon to investigate the
antagonistic or potentiating role that the microbiome has in tumor formation.
These models will enable us to determine whether there are specific microbial
populations or interactions that have a causal role or serve as biomarkers of
tumorigenesis. To address these questions we use a variety of
culture-independent techniques focused on DNA sequencing of the 16S rRNA gene as
well as bulk bacterial DNA (aka metagenomics). These tools enable us to address
our biological questions from a taxonomic and functional perspective.
Furthermore, through the use of mathematical modeling we hope to improve our
understanding of the mechanistic underpinnings of the microbiome as well as
predict the effects of perturbations on the microbiome.


### Microbial informatics
Because we are at the forefront of applying modern sequence-based tools to
biological questions, we are also developing software to describe and compare
microbial communities. This software enables the scientific community to reach
conclusions regarding host-microbiome interactions with statistical confidence
and has effectively enabled the field to transition from discover-based to
hypothesis-driven research. As part of this effort, we provide workshops to
train microbial ecologists on the theory and practice of analyzing their data.
Our program, mothur (http://www.mothur.org), is being widely used by microbial
ecologists that have generated Sanger sequence data and pyrosequence tag
sequences. We also collaborate with other investigators to conduct studies in
murine, human, and "environmental" samples to help design experiments and
analyze data related to studying microbial ecology.


### Sound interesting?
We are always on the lookout for excellent students and postdoctoral fellows. My
goal is to train scientists to straddle the fields of microbiology, ecology, and
bioinformatics. Attempting to study host-microbiome interactions without this
approach is limiting and so I believe that although this course of training is
demanding, it will prepare people for the next generation of science.
